Processes/DevRel-Publication-Process.md

```diff
@@ -0,0 +1,169 @@
+# Process: Publish New or Updated Content to the Dev Portal
+
+## Overview
+This document provides instructions on how to publish new or updated content to the EOS Dev Portal (docs.eosnetwork.com).
+
+**Note**: New or updated content is published to a private staging environment every day, which allows for content review prior to publication of content to the public dev portal. Once content is reviewed and approved for public consumption, the content can be published to the public dev portal. This publication process can be scheduled or ad hoc, depending on the need for the community to consume the content.
+
+### When to Use this Procedure
+Use this procedure when you need to update content from multiple repositories to the [EOS Dev Portal](http://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption on the EOS Dev Portal.
+
+# Procedures Contained in This Publication Process
+Complete the following procedures to publish new or updated content to the EOS Dev Portal:
+1. Create/Edit Files
+2. Create Commits/Push Changes
+3. Check Changes on Your Remote/Target Branch
+4. Pull Remote Changes from Your Remote Branch
+5. Update Existing Files (Refer to Create/Edit Files)
+6. Rename Files
+
+## Definitions
```

Comment:
Please remove this part, it isn't this doc's job to teach its readers Git/GitHub

You can replace it with a comment about our main branch being called `main`, and an external link to learning Git/GitHub if you like

Reply:
Done

Reply:
Agreed, this procedure should be simpler. However, these details are still useful for someone onboarding to the DevRel team. You could move the original document to a new "Onboarding" folder, for instance, and lay out simpler steps for this "How to Contribute Content to the EOS Documentation" document.